api/build.gradle

```diff
@@ -20,23 +20,26 @@ buildscript {
     // External properties on the default project. Values declared in ext blocks
     // outside of the buildscript block aren't usable here.
     ext {
-        GAE_VERSION = '2.0.29'
+        GAE_VERSION = '2.0.31'
         GOOGLE_TRUTH_VERSION = '1.1.3'
         GSON_VERSION = '2.10'
         HIBERNATE_VERSION = '6.5.2.Final'
         JACKSON_VERSION = '2.17.2'
-        LIQUIBASE_VERSION = '4.29.0'
-        LOGBACK_VERSION = '1.5.6'   // upgrading past this results in an error
-        MAPSTRUCT_VERSION = '1.5.5.Final'
+        LIQUIBASE_VERSION = '4.29.2'
+        // 1.5.7+ results in a runtime error:
+        // Logging system failed to initialize using configuration from 'null'
+        //  java.lang.NoSuchMethodError: 'java.lang.Object ch.qos.logback.classic.LoggerContext.getConfigurationLock()'
+        LOGBACK_VERSION = '1.5.6'
+        MAPSTRUCT_VERSION = '1.6.2'
         OKHTTP_VERSION = '4.12.0'
         OPENTELEMETRY_SDK_VERSION = '1.43.0'
-        OPENTELEMETRY_JAVA_VERSION = '2.5.0'
+        OPENTELEMETRY_JAVA_VERSION = '2.8.0'
         OPENTELEMETRY_GOOGLE_VERSION = '0.31.0'
         SPRINGFOX_VERSION = '3.0.0'
         SPRING_BOOT_VERSION = '3.3.2'
-        SPRING_FRAMEWORK_VERSION = '6.1.9'
-        SPRING_SECURITY_VERSION = '6.3.0'
-        SWAGGER_3_CODEGEN_VERSION = '3.0.55'
```

Comment:
this was only used in one location, so I inlined it.  `SWAGGER_VERSION` refers to other packages.